addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1622,6 +1648,27 @@ MockServer.include({
         ]);
         this._mockUnlink(model, [followers.map(follower => follower.id)]);
     },
+    /**
+     * Simulates `set_mail_user_settings` on `mail.user.settings`.
+     *
+     * @param {integer} id
+     * @param {object.<string, boolean>} new_settings
+     */
+     _mockMailUserSettingsSetMailUserSettings(id, new_settings) {
+        this._mockWrite('mail.user.settings', [
+            [id],
+            new_settings,
+        ]);
+        const mailUserSettings = this._getRecords('mail.user.settings', [['id', '=', id]]);
+        const notification = [
+            ['dbName', 'res.partner', this.currentPartnerId],
+            {
+                type: 'mail_user_settings',
+                payload: mailUserSettings,
```

Comment:
same as in server, only send what changed

Reply:
done